Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -27,6 +28,21 @@ protected void Update()
         {
             //TODO: Not sure if this works with 2 cameras?? Wait until later (using x negative scaling for now)
             interactTransform.LookAt(Camera.main.transform);
+
+            // Ensures player game object has been assigned
+            if (player is null)
+            {
+                Debug.Log("Player refernce not found");
```

Comment:
Maybe change this to `Debug.LogError`
Also maybe this check should be in `FaceInteractable` not in the update method? (if so, try to do an early exit if `player is null` by returning early)